packages/components-js/tests/e2e/specs/link-tile-product.e2e.ts

```diff
@@ -46,43 +46,51 @@ test.describe('like button', () => {
     await initLinkTileProduct(page);
     const host = getHost(page);
     const likeButton = getLikeButton(page);
-    expect(likeButton).toBeDefined();
-    expect(await getProperty(host, 'liked')).toBe(false);
-    expect(await getProperty(likeButton, 'icon')).toBe('heart');
+
+    await expect.poll(() => likeButton).toBeDefined();
+    await expect.poll(async () => await getProperty(host, 'liked')).toBe(false);
+    await expect.poll(async () => await getProperty(likeButton, 'icon')).toBe('heart');
```

Comment:
Instead of `poll` we can change how the expectations are done.
E.g.:
```suggestion
    await expect(likeButton).toBeDefined();
    await expect(host).toHaveJsProperty('liked', false);
    await expect(likeButton).toHaveJsProperty('icon', 'heart');
```

Reply:
changed to suggestion, besides ` await expect(likeButton).toBeDefined();` which did not work.